sourcegraph/sourcegraph
# Add initial onboarding (public) repos

Closes https://github.com/sourcegraph/sourcegraph/issues/42982.
## Test plan
- Remove `EXTSVC_CONFIG_FILE` env var
- `ONBOARDING_EXTERNAL_SERVICE_GH_TOKEN={GitHub dummy personal access token} sg start`
- Open http://localhost:3080/site-admin/external-services
- Check that a new external service was added
- Rerun (or also delete external service) and make sure that no new external services were added
